packages/common/src/items/follow.ts

```diff
@@ -0,0 +1,140 @@
+import { IUser, joinGroup, leaveGroup } from "@esri/arcgis-rest-portal";
+import { HubEntityType, fetchHubEntity } from "../core";
```

Comment:
Just since I've gotten these comments before, we should update this file path to go all the way to the file instead of stopping at `../core`